data/espers.py

```diff
@@ -345,7 +345,7 @@ def get_random_esper(self):
         if not self.available_espers:
             return None
 
-        rand_esper = random.sample(self.available_espers, 1)[0]
+        rand_esper = random.sample(list(self.available_espers), 1)[0]
```

Comment:
```suggestion
        rand_esper = random.sample(sorted(self.available_espers), 1)[0]
```
The order of iterating through a set is not deterministic.
If we want to make sure it gets the same result every time with a given seed, we have to make sure the order is the same every time.

Reply:
thanks, changed